core/src/main/java/io/apicurio/hub/api/codegen/OpenApi2JaxRs.java

```diff
@@ -568,6 +574,23 @@ protected String generateJavaInterface(CodegenInfo info, CodegenJavaInterface in
                     default:
                         break;
                 }
+
+                boolean forbidNotNull = 
+                        paramType.isPrimitive() ||
+                        "path".equals(arg.getIn()) ||
+                        !arg.getRequired();
+
+                addValidationConstraints(param, arg, forbidNotNull, topLevelPackage);
+
+                Optional.ofNullable(arg.getAnnotations())
```

Comment:
If I understand this correctly we are re-parsing the code generated from `jsonschema2pojo` and applying transformation to it, this might be extremely expensive for large APIs.
Have we tested how much those changes are affecting the overall speed?

I know I'm super-late at the party for providing an alternative to `jsonschema2pojo` 😞 , but, should we do it before adding this functionality?